modules/javafx.base/src/main/java/javafx/beans/property/BooleanProperty.java

```diff
@@ -180,4 +180,4 @@ public String getName() {
             }
         };
     }
-}
+}
```

Comment:
Minor: can you add the trailing newline back in? We typically end each file with a single new-line; also this shows up as an unnecessary diff.